spot-contracts/contracts/vaults/RolloverVault.sol

```diff
@@ -163,11 +177,12 @@ contract RolloverVault is
 
     /// @inheritdoc IVault
     /// @dev Its safer to call `recover` before `deploy` so the full available balance can be deployed.
-    ///      Reverts if there are no funds to deploy.
+    ///      Reverts if no funds are rolled over or if the minimum deployment threshold is not reached.
     function deploy() public override nonReentrant whenNotPaused {
-        TrancheData memory td = _tranche(perp.getDepositBond());
-        if (_rollover(perp, td) <= 0) {
-            revert NoDeployment();
+        (uint256 deployedAmt, TrancheData memory td) = _tranche(perp.getDepositBond());
+        uint256 perpsRolledOver = _rollover(perp, td);
+        if (deployedAmt <= minDeploymentAmt || perpsRolledOver <= 0) {
```

Comment:
```suggestion
        // NOTE: The following enforces that we only tranche the underlying if it can immediately be used for rotations.
        if (deployedAmt <= minDeploymentAmt || perpsRolledOver <= 0) {
```